{"name": "testcaseadd3[【类型】输入特殊字符，给出提示信息2]", "status": "passed", "steps": [{"name": "步骤一：21.45和@开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\practice\\study\\study0824\\testcase\\test_add.py\", line 48, in testcaseadd3\n    result = self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1661497761884, "stop": 1661497761884}], "attachments": [{"name": "log", "source": "dffcca08-47b3-4ed1-b68a-b989dc43768e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "21.45"}, {"name": "y", "value": "'@'"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1661497761883, "stop": 1661497761884, "uuid": "7b7484fe-7c33-44a1-bde1-baeee4074883", "historyId": "a97120f1741555fb5a39b0b96ba40382", "testCaseId": "2f8f937ca82ff540c8b5f46a0207716d", "fullName": "practice.study.study0824.testcase.test_add.Testadd#testcaseadd3", "labels": [{"name": "story", "value": "加法模块测试error"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "practice.study.study0824.testcase"}, {"name": "suite", "value": "test_add"}, {"name": "subSuite", "value": "Testadd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "33120-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "practice.study.study0824.testcase.test_add"}]}